### PR TITLE
Various fixes

### LINKED ISSUE #175: Delete in text field
When pressing delete in a text field in order to delete text, the active shot is deleted

### PR DESCRIPTION
**Pull request by mennoo1996**
Fixes #175 
Removed unneeded print statements
Fixed director shot add modal height so all buttons are displayed
Camera type add modal had wrong title label, fixed
Delete in text field no longer removes shot
Director shots are now deleted on new project (this did not happen before)

### CODE REVIEW

File: src/main/java/control/ToolViewController.java [hunk 1]
```diff
@@ -56,21 +56,48 @@ private void initializeTools() {
...
         // Add Delete Key Event Listener for deleting active shot
         this.controllerManager.getRootPane().getPrimaryStage()
-                .getScene().addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+                .getScene().addEventFilter(KeyEvent.ANY, event -> {
                         if ((event.getCode() == KeyCode.DELETE)
                                 || (event.getCode() == KeyCode.BACK_SPACE
                                     && event.isShortcutDown())) {
-                            deleteActiveCameraShot();
-                            event.consume();
+                            System.out.println(this.controllerManager.getRootPane()
```
Comment (bart-de-jonge) [inline src/main/java/control/ToolViewController.java]: Please remove this syso. Maybe use the logger here

**Comment by bart-de-jonge** [comment]
One small comment

File: src/main/java/control/ToolViewController.java [hunk 2]
```diff
@@ -56,21 +56,45 @@ private void initializeTools() {
...
+                                    currentFocusClass.equals("gui.styling.StyledTextfield")
+                                    ||
+                                    currentFocusClass.equals("javafx.scene.control.TextField")
+                                    ||
+                                    currentFocusClass.equals("gui.headerarea.DoubleTextField")
+                                    ||
+                                    currentFocusClass.equals("gui.headerarea.NumberTextField");
+                            System.out.println(isTextField);
+                            if (isTextField) {
+                                //event.consume();
```
Comment (Markvanderuit) [inline src/main/java/control/ToolViewController.java]: Dead code?

**Comment by Markvanderuit** [comment]
One tiny comment also. Fix/answer those, and we'll merge. 👍

File: src/main/java/control/ToolViewController.java [hunk 3]
```diff
@@ -56,21 +56,43 @@ private void initializeTools() {
...
         // Add Delete Key Event Listener for deleting active shot
         this.controllerManager.getRootPane().getPrimaryStage()
-                .getScene().addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+                .getScene().addEventFilter(KeyEvent.ANY, event -> {
                         if ((event.getCode() == KeyCode.DELETE)
                                 || (event.getCode() == KeyCode.BACK_SPACE
                                     && event.isShortcutDown())) {
-                            deleteActiveCameraShot();
-                            event.consume();
+                            String currentFocusClass = this.controllerManager.getRootPane()
```
Comment (martijnjanssen) [inline src/main/java/control/ToolViewController.java]: Just place the entire part after the `=` on the new line.

File: src/main/java/control/ToolViewController.java [hunk 4]
```diff
@@ -56,21 +56,43 @@ private void initializeTools() {
...
+                                               .getScene().getFocusOwner().getClass().getName();
+                            
+                            boolean isTextField = 
+                                    currentFocusClass.equals("gui.styling.StyledTextfield")
+                                    ||
+                                    currentFocusClass.equals("javafx.scene.control.TextField")
+                                    ||
+                                    currentFocusClass.equals("gui.headerarea.DoubleTextField")
+                                    ||
+                                    currentFocusClass.equals("gui.headerarea.NumberTextField");
```
Comment (martijnjanssen) [inline src/main/java/control/ToolViewController.java]: The `||` parts should be before the line itself, or is the line too long?

**Pull request merged**